project/plugins.sbt

```diff
@@ -33,7 +33,7 @@ addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "2.2.0")
 lazy val ow2Version = "9.5"
 lazy val jacocoVersion = "0.8.11-absa.1"
 
-def jacocoUrl(artifactName: String): String = s"https://github.com/AbsaOSS/jacoco/releases/download/$jacocoVersion/org.jacoco.$artifactName-$jacocoVersion.jar"
+def jacocoUrl(artifactName: String): String = s"file:///C:/Users/ABLL526/Documents/GitHub/Jar-Store2/org.jacoco.$artifactName-$jacocoVersion.jar"
```

Comment:
???

Reply:
Yes, this was for my use only. Will remove the file from the PR.